blog/2020-05-11-version-0.5.0-released.md

```diff
@@ -0,0 +1,86 @@
+---
+id: version-0.5.0-released
+title: Version 0.5.0 released
+author: Beanow
+author_url: https://github.com/Beanow
+author_image_url: https://avatars3.githubusercontent.com/u/497556?v=4&s=200
+---
+
+It's been months since the last versioned release, so there's a lot new to
+v0.5.0!
+
+You can find the release [on GitHub][GitHub release] or as a [Docker image].
+More details on how to use them can be found in the [installation guide].
+
+Since this is a large release, I'll only touch on several major additions here.
+
+### Discourse plugin
+
+Discourse forums can be analyzed with this release. And from using it so far, I
+think it's a really interesting plugin.
+
+Discourse forums by default require no authentication to be set up. Meaning you
+can load most Discourse forums out there and explore them with SourceCred, no
+approval required. The [plugin setup guide][discourse setup] walks through the
+commands to do this.
+
+Much thanks to the straightforward 💚 likes on Discourse, the out-of-the-box
+settings tend to be quite reasonable. Most of the tweaking involves balancing
+the Discourse weight relative to other platforms.
+
+### Initiatives
+
+[Initiatives] are added to SourceCred by a new plugin of the same name.
+Instructions are in it's [setup guide][initiatives setup].
+
+While being technical to use in it's current version, it adds a number of new
+capabilities. Such as manually adding contributions for which there's no plugin,
+incentivizing work before it's completed and grouping contributions together to
+refer to them more easily.
+
+Since it's a recent addition, the jury is still out on whether we like the
+default settings and what the best ways to use it are. There's a good chance
+it will evolve as we test it out for ourselves and receive feedback from you.
+
+### Docker image
+
+As mentioned at the beginning of this post, SourceCred is being released as
+Docker images as well. They are available as [`sourcecred/sourcecred` on
+DockerHub][Docker image].
+
+The tags to be aware of are:
+- `latest` (default) which refers to the most recent versioned release.
+- `vX.X.X` versioned releases, which allow you to use a specific version.
+- `dev` which corresponds to the latest `master` branch.
+
+Even though this is the first major version to be included, the images are
+already seeing regular use by the community. Especially because it's a great way
```

Comment:
"... by the community, especially because..."
This keeps it from being a sentence fragment

Reply:
Thanks, going with the suggested comma here :]
As sentence fragment fixing practice, tried to do the same as the above and turn them into independent sentences, but the ones I came up with felt like contrived fluff.